airbyte-commons/src/main/resources/seed/specs_secrets_mask.yaml

```diff
@@ -0,0 +1,114 @@
+properties:
+- apikey
+- rest_credential
+- app_secret_token
+- aws_access_key
+- auth_token
+- extension_id
+- openai_key
+- key
+- accessKey
+- api_token
+- account_sid
+- client_secret
+- azure_blob_storage_sas_token
+- refresh_token
+- motherduck_api_key
+- hmac_key_access_id
+- consumer_key
+- X-Postmark-Server-Token
+- service_account
+- api_key
+- application_key
+- password
+- api_key_secret
+- api_password
+- service_account_info
+- google_application_credentials
+- public_key
+- privateKey
+- realm
+- access_key_id
+- aws_key_secret
+- databricks_personal_access_token
+- secret_key
+- secret
+- pinecone_key
+- aws_access_key_id
+- key_store_password
+- user_token
+- app_id
+- survey_group_id
+- realm_id
+- X-Postmark-Account-Token
+- apiKeySecret
+- ssl_certificate
+- sas_token
+- secret_access_key
+- shared_key
+- client_secret_value
+- client_id
+- uid
+- service_account_json
+- survey_id
+- rest_token
+- hmac_key_secret
+- reserved_attribute_names
+- appid
+- tenant_id
+- aws_key_id
+- role_arn
+- account_id
+- ca_certificate
+- access_key_secret
+- credentials_json
+- user_name
+- private_key_password
+- tunnel_user_password
+- apiKey
+- auth_user_password
+- s3_secret_access_key
+- api_id
+- org_id
+- private_key
+- token_secret
+- ssh_key
+- consumer_secret
+- api_secret
+- access_token
+- developer_token
+- user_auth_key
+- site_api_key
+- client_certificate
+- account_token
+- access_key
+- agreement_grant_token
+- apiId
+- jwt_token
+- token
+- lwa_app_id
+- aws_secret_access_key
+- session_token
+- personal_access_token
+- client_key_password
+- azure_blob_storage_account_key
+- value
+- aws_secret_key
+- restid
+- user_secret
+- auth_hash
+- client_key
+- cohere_key
+- ssl_ca_certificate
+- app_api_key
+- sasl_jaas_config
+- key_encrypting_key
+- token_key
+- site_id
+- token_uri
+- s3_access_key_id
+- auth_ssh_key
+- application_secret
+- lwa_client_secret
+- domain_url
+- storage_access_key
```

Comment:
Are these other file changes on purpose?

Reply:
No, I don't know how that ended in my PR